### ISSUE TITLE
Wrong number of Database queries

### ISSUE THREAD

**Issue by nitriques** [symphonycms/profiledevkit#6]
I do not know why, but I have contradictory stats.

In render-stats, I get 352 database queries, but when I go into data base queries, I only see 275.

Anyone has ever observed this ?

**Comment by nitriques** [comment]
BTW on my test setup, it's off by only one (36 vs. 37)

**Comment by brendo** [comment]
> In render-stats, I get 352 database queries, but when I go into data base queries, I only see 275.

Can you confirm you're not using any caching that would cause the huge drops? Do you have any curious extensions on this installation? There is no reason why there should be a discrepancy between the two pages.

> BTW on my test setup, it's off by only one (36 vs. 37)

Is the one query the Session cleanup being triggered?

**Comment by nitriques** [comment]
> Can you confirm you're not using any caching that would cause the huge drops? 

Nothing that I am aware of. I do have memcached on that server but did not configured anything.

> Do you have any curious extensions on this installation? 

I do not think so. Maybe ERF (?) But it's only a field... 

> Is the one query the Session cleanup being triggered?

The only query I see related to the session is this one

```sql
SELECT SQL_CACHE `session_data` FROM `sym_sessions` WHERE `session` = '<token>' LIMIT 1
```

**Comment by ChoboHub** [comment]
It seems like it is fix in 2.7.x on PHP 7 (on localhost) I get same value : 

Total Database Queries  27

27  0.0005  SELECT SQL_CACHE \* FROM `sym_tracker_activity` WHERE `item_type` REGEXP "[[:digit:]]+" AND (`action_type` = "created" OR `action_type` = "updated") ORDER BY `timestamp` desc LIMIT 0, 20

Sometimes I go to 28 (because of a DELETE) but both follow the same numbers of queries.

Tried couple of times, added new sections and ERF but it's working well.
